.cirrus.yml

```diff
@@ -50,6 +50,28 @@ task:
     - bash ci/script.sh
   before_cache_script: rm -rf $CARGO_HOME/registry/index
 
+
+# macOS aarch64 requires a more recent toolchain than the other macOS targets.
+task:
+  name: macOS aarch64
+  env:
+    TARGET: aarch64-apple-darwin
+    TOOLCHAIN: stable
```

Comment:
Can you specify an exact rustc version number instead of "stable"?

Reply:
I've set it to 1.50 since it requires a new version, although I'm not sure how new.

Reply:
Could you please find out?